### DISCUSSION TITLE
Problems compiling the docker image with pre-fab GPU box

### DISCUSSION THREAD

**Discussion by AleaScovill** [opendr-eu/opendr#272] [Problems and questions]
There was a problem with compiling the docker image for the tx2. There is software pre-flashed onto the gpu that's causing a conflict.

This is the link to the box that:
http://www.syslogic.com/eng/ai-vehicle-computer-jetson-tx2-102493.shtml

**Comment by passalis** [comment]
This has been resolved in the final version.
